Sources/BraintreePayPalNativeCheckout/BTPayPalNativeOrderCreationClient.swift

```diff
@@ -33,11 +33,13 @@ class BTPayPalNativeOrderCreationClient {
             }
 
             guard config.json["paypalEnabled"].isTrue else {
+                self.apiClient.sendAnalyticsEvent("ios.paypal-native.create-order.paypal-not-enabled.failure")
```

Comment:
See [naming guide](https://github.braintreeps.com/team-sdk/sdk-knowledge/blob/main/analytics/events/event-names.md), prefer past tense such as `failed`.

Reply:
Also similar comment here regarding just needing `ios.paypal-native.create-order. succeeded` and `...failed` cases.

Reply:
Gotcha - can update this, had not seen this before, was just going off of venmo which uses success/failure

Reply:
Sounds good!

Yeh, there has been a lot of inconsistency with naming. We did an audit awhile ago and looked for the most common naming convention used and decided we should try to use that going forward. Re-naming existing analytics isn't usually worth it though because then we have 2 event names in the wild that mean the same thing, and that gets 🤯.

Reply:
Ah, that makes sense - I am glad we have a guide for this!